core/common/modules/finder/assets/js/component.js

```diff
@@ -1,54 +1,12 @@
-import ComponentModalBase from 'elementor-api/modules/component-modal-base';
-import FinderLayout from './modal-layout';
+import ComponentBase from 'elementor-api/modules/component-base';
+import NavigateComponent from './navigate/component';
 
-export default class Component extends ComponentModalBase {
+export default class Component extends ComponentBase {
```

Comment:
What happen here? (navigate is not a separated component, if needed - rename `navigate/down` to `navigate-down` + BC)

Reply:
Fixed here: https://github.com/elementor/elementor/pull/10868